distribution/docs/src/main/resources/content/_sources/solr-catalog-provider.adoc

```diff
@@ -32,13 +32,6 @@ No installation is required because ${branding} includes a distribution of Apach
 .Configuring Solr Server
 No configuration.
 
-.Solr Server System Properties (default values)
- ----
- start.solr=true
- solr.client=HttpSolrClient
- solr.http.port=8994
- ----
```

Comment:
It was not a good idea to document the settings both here and in global-setting.adoc. Removed.